lib/sles4sap/ipaddr2.pm

```diff
@@ -1915,4 +1952,245 @@ sub ipaddr2_test_other_vm {
     }
 }
 
+=head2 ipaddr2_clean_network_peering
+
+    ipaddr2_clean_network_peering(ibsm_rg => );
+
+Cleanup the network peering if needed.
+
+=over
+
+=back
+
+=cut
+
+sub ipaddr2_clean_network_peering {
+    my (%args) = @_;
+    croak 'Missing mandatory argument < ibsm_rg >' unless $args{'ibsm_rg'};
+    qesap_az_vnet_peering_delete(source_group => ipaddr2_azure_resource_group(), target_group => $args{'ibsm_rg'});
+}
+
+=head2 ipaddr2_network_peering
```

Comment:
Missing verb in the function name `ipaddr2_network_peering` --> `ipaddr2_network_peering_create`